readthedocs/source/developer_installation.rst

```diff
@@ -21,8 +21,8 @@ To install CCL using a ``pip`` developer installation, you can execute
 from the top-level directory in the repository. You will need ``CMake`` in
 order to install CCL in this way. See :ref:`getting-cmake` for help installing
 ``CMake`` if you do not already have it. In order to run the tests,
-you will need both ``CAMB`` and ``CLASS`` installed. See the instructions for
-:ref:`boltzmann-codes` for details.
+you will need ``CAMB``, ``CLASS``, and ``FastPT`` installed. See the instructions for
```

Comment:
e.g. here. In general, it's best to have everything capitalized: FAST-PT

Reply:
Capitalised FAST-PT now.